src/link_prober/IcmpPayload.h

```diff
@@ -42,19 +42,76 @@ __BEGIN_DECLS
 
 namespace link_prober
 {
+/**
+ *@enum TlvType
+ *
+ *@brief Supported TLV types
+ */
+enum TlvType: uint8_t {
+    TLV_COMMAND = 1,
+    TLV_SENTINEL = 0x0f,
+    TLV_COUNT
+};
+
+#define TlvHead      \
+    uint8_t type;    \
+    uint16_t length
```

Comment:
Why not define `TlvHead` as a new struct?

Reply:
I use MACRO here because it will not introduce an extra layer